ApiEverest/CrudApi/ApiEverest.csproj

```
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+	<PropertyGroup>
+		<TargetFramework>net6.0</TargetFramework>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Swashbuckle.AspNetCore" Version="6.4.0" />
+	</ItemGroup>
+
+	<ItemGroup>
+	  <ProjectReference Include="..\AppServices\AppServices.csproj" />
```

```suggestion
	  <ProjectReference Include="..\AppServices\AppServices.csproj" />
```
Corrigir indentação com CTRL + K + D